installer/charts/rhtap-acs/templates/tests/test.yaml

```diff
@@ -32,6 +32,11 @@ spec:
           value: {{ $namespace }}
         - name: RESOURCE_TYPE
           value: "deployment"
+      {{- if eq . "scanner" }}
+        # Downloading the vulnerability DB can take a long time
```

Comment:
By default we allow a 15 minute timeout. Should we make sure the Helm Chart installation waits longer than that?

Reply:
My tests shows that 15m is enough for the test to pass, we should be OK.

Reply:
Another test shows that the 15m Helm timeout does not apply to tests. I'm surprised by that result. There is a timeout flag (https://helm.sh/docs/helm/helm_test/#options), I wonder if we use it at all.